starwars/src/components/CharicterCards.js

```diff
@@ -0,0 +1,75 @@
+import axios from "axios"
+import styled from 'styled-components';
+
+import React, {useState, useEffect} from 'react';
+// import Films from "./Films.js"
+const Cards = styled.div`
+    display: flex;
+    align-items: center;
+    align-items: space-evenly;
+    font-family: nasalization, sans-serif;
+    flex-flow: column;
+
+
+    }`
+    const Card = styled.div`
+font-family: nasalization, sans-serif;
+    display: flex;
+    justify-content: center;
+
+    border-top: 2px solid #FFE5C0;
+    border-bottom: 2px solid #FF3C5C;
+    margin-bottom: 5px;
+    width: 75%;
+        }`
+const Person = styled.h1`
+    display: flex;
+    flex-flow: column;
+
+    justify-content: center;
+    align-items: space-evenly;
+    text-shadow: 0 0 3px #0A0000, 0 0 5px #FF3C5C;
+    color: #FFE5C0;
+    }`
+    const Birth = styled.p`
+      color: #FFA72B;
+        }`
+        const Homeworld = styled.button`
+          color:  #17A694;
+          border: none;
+          background: none;
+          font-family: nasalization, sans-serif;
+          text-decoration: none;
+          font-size: 40px;
+            }`
+
+export default function CharicterCards(props) {
```

Comment:
technically speaking, CharicterCards is the highest level component that is requiring the data so it's fine to pull the data here, but on a smaller app like this, my opinion is that it's easier to read if you put the useEffect/axios inside the app.js and then pass down the data as props.  The idea would be similar to how we like styled-components for being able to put css with the JSX in the same file.